lib/config/go_routes/go_router.dart

```diff
@@ -431,6 +432,14 @@ abstract class AppRoutes {
             ),
           ],
         ),
+        GoRoute(
+          path: '/share',
+          pageBuilder: (context, state) => defaultPageBuilder(
+            context,
+            const Share(),
```

Comment:
`ShareWidget` or `ShareScreen`

Reply:
ahhh. but still miss a lot of information

Reply:
it's like `Chat`